Tools/clinic/clinic.py

```diff
@@ -1502,13 +1361,17 @@ def parser_body(
                 if p.deprecated_keyword:
                     deprecated_keywords[i] = p
 
-            has_optional_kw = (max(pos_only, min_pos) + min_kw_only < len(converters) - int(vararg != NO_VARARG))
+            has_optional_kw = (
+                max(pos_only, min_pos) + min_kw_only
+                <
+                len(converters) - int(vararg != self.NO_VARARG)
```

Comment:
PEP-8: https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

```suggestion
                < len(converters) - int(vararg != self.NO_VARARG)
```